tarynmartin/shopify-challenge
# Feature/local storage

#### What’s this PR do?
- Allows a user to remember their nominations when they return to the site
- Clears nominations when the array is at 0

#### Where should the reviewer start?
- App.js - look at setLocalStorage, first & third useEffect functions to see how localStorage is working

#### How should this be manually tested?
- in terminal, run npm start
- in browser, nominate several movies & refresh page to see nominations stay
- delete some, refresh, and see changes continue

#### What are the relevant tickets?
resolves #10
